### PR TITLE
Fix calendar tests

### PR DESCRIPTION
**Pull request by Var1377**

### CODE REVIEW

**Review by SamYap-Softwire — APPROVED** [review]
Seems good. Thanks.

File: src/components/Calendar/Calendar.cy.tsx [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React from "react";
 import Calendar, { CalendarEvent } from "@/components/Calendar/Calendar";
 
 describe("<Calendar />", () => {
-    const today = new Date();
-    const tomorrow = new Date();
+    const today = new Date("2021-04-05");
```
Comment (TomCusack-Softwire) [inline src/components/Calendar/Calendar.cy.tsx:5]: can you rename these to be more accurate, e.g. `testDate` and `dayAfterTestDate`?

File: src/components/Calendar/Calendar.cy.tsx [hunk 2]
```diff
@@ -34,33 +34,33 @@ describe("<Calendar />", () => {
...
     it("can change view to timeGridWeek", () => {
-        cy.mount(<Calendar initialEvents={[]} />);
+        cy.mount(<Calendar initialEvents={[]} initialDate={today} />);
         cy.get("button.fc-timeGridWeek-button").click();
         cy.get(".fc-timeGridWeek-view").should("be.visible");
     });
 
     it("can change view between months in dayGridMonth", () => {
-        const prevMonth: Date = new Date();
+        const prevMonth: Date = new Date(today);
```
Comment (TomCusack-Softwire) [inline src/components/Calendar/Calendar.cy.tsx:59]: can you get rid of `: Date` type annotation?
apologies, I know this was already on main and not a change you made, just now that I've noticed it it's going to annoy me!

**Review by TomCusack-Softwire — APPROVED** [review]
Capital letter to start the PR (Fix)
Otherwise looks good! Thanks for debugging :) Feel free to merge when minor stylistic comments below are fixed!

**Pull request merged**